specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchindex.json

```diff
@@ -1240,6 +1236,7 @@
           },
           "readOnly": true,
           "x-ms-client-name": "Captions",
+          "x-nullable": true,
```

Comment:
Note: I see this was added, so maybe that answers my question I re-activated, i.e. perhaps all those properties would not return an explicit `null`.